src/dune_rules/global.ml

```diff
@@ -9,7 +9,7 @@ let init ~capture_outputs =
          (not capture_outputs)
          || not (Lazy.force Ansi_color.stderr_supports_color)
```

Comment:
All of this logic should move to the colors module as well. `Global.init` should just take an environment.